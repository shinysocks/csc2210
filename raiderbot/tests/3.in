+--------------------+  
|    **# ########  | |
|  # ** #*    *    --|
| #  ** =*...*#*     |
| #     #*    *      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
19 0   
www
swww
esswwn
wwwwns
wwwn
wwn
esseeee  
ssws   
