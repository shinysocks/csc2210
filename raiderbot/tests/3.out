Initial state: Robot at 19, 0 (0 gold)
+--------------------+
|    **# ########  |R|
|  # ** #*    *    --|
| #  ** =*...*#*     |
| #     #*    *      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
========================================
Executing www
Robot at 16, 0 (0 gold)
+--------------------+
|    **# ########R | |
|  # ** #*    *    --|
| #  ** =*...*#*     |
| #     #*    *      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
Executing swww
Robot at 13, 1 (1 gold)
+--------------------+
|    **# ########  | |
|  # ** #*    R    --|
| #  ** =*...*#*     |
| #     #*    *      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
Executing esswwn
Robot at 12, 2 (4 gold)
+--------------------+
|    **# ########  | |
|  # ** #*         --|
| #  ** =*...R#      |
| #     #*           |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
Executing wwwwns
Robot at 8, 2 (6 gold)
+--------------------+
|    **# ########  | |
|  # ** #          --|
| #  ** =R... #      |
| #     #*           |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
Executing wwwn
Robot at 5, 1 (8 gold)
+--------------------+
|    **# ########  | |
|  # *R #          --|
| #  *  = ... #      |
| #     #*           |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
Executing wwn
Robot at 3, 0 (9 gold)
+--------------------+
|   R**# ########  | |
|  #    #          --|
| #  *  = ... #      |
| #     #*           |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
Executing esseeee
Robot at 8, 2 (11 gold)
+--------------------+
|     *# ########  | |
|  #    #          --|
| #     =R... #      |
| #     #*           |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
Executing ssws
Robot at 7, 5 (14 gold)
+--------------------+
|     *# ########  | |
|  #    #          --|
| #     = ... #      |
| #     #            |
|  #   #     #B#     |
|       R    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
========================================
Robot has completed its task.
